src/Application/ConfigResolver.php

```diff
@@ -63,6 +63,10 @@ public static function guess(string $directory): string
 
         $composer = json_decode((string) file_get_contents($composerPath), true);
 
+        if (empty($composer['require']) || !is_array($composer['require'])) {
```

Comment:
I think you should adapt each preset with a rule that if require don't exists, we return `false` instead of having this.

Reply:
Allright

Reply:
Reason is: Some presets return true, even if the `composer` don't have `require` key.

Reply:
Yes, I just saw that :sweat_smile: Fixed